### PR TITLE
prep-html-syntax - Simon Kim

### PR DESCRIPTION
**Pull request by Skim-329**
Screen Shot 2021-07-31 at 4 14 39 PM

### CODE REVIEW

File: prep-html-syntax/index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <h1 style="color: green;">My favorite Star Wars character is Yoda!
+    </h1>
+    <img src="https://img.money.com/2019/11/baby-yoda-toys-for-sale-1163281433.jpg?quality=60&w=800" alt="Yoda"
+  <br>
+    <p style="color: rgb(104, 68, 22);">Yoda may look frail but he is wise and powerful!
+    </p>
+  </body>
```
Comment (thebearingedge) [inline prep-html-syntax/index.html:13]: Hi Simon, be sure to divide your page into two specific sections. The `<head>` should open and close before the `<body>` opens and closes. The visible content of the page should go in the `<body>`.

```html
<!DOCTYPE html>
<html>
  <head>
    <!-- meta info like <title></title> goes in here -->
  </head>
  <body>
    <!-- visible content like <h1></h1> and <p></p> goes in here -->
  </body>
</html>
```

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: prep-html-syntax/index.html (hunk 1 above)
Comment (Skim-329) [reply prep-html-syntax/index.html:13]: Hey Tim, thanks for the clarification. By <head> are you referring to the <h1> that I used? Also, to make a change, should I just begin the assignment again?

Comment (Skim-329) [reply prep-html-syntax/index.html:13]: *h1 designation.

**Review by Skim-329 — COMMENTED** [review]
Screen Shot 2021-08-02 at 9 55 02 PM

File: prep-html-syntax/index.html (hunk 1 above)
Comment (thebearingedge) [reply prep-html-syntax/index.html:13]: Hi @Skim-329 I was just referring to the missing `<head>` element. Looks good now!

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**